subsys/net/l2/ethernet/ethernet.c

```diff
@@ -34,6 +34,18 @@ const struct net_eth_addr *net_eth_broadcast_addr(void)
 	return &broadcast_eth_addr;
 }
 
+#if defined(CONFIG_NET_VLAN)
+static u8_t eth_hdr_buffer[sizeof(struct net_eth_vlan_hdr)];
```

Comment:
Please call something like "eth_hdr_data".

The changes in this file (l2/ethernet/ethernet.c) don't belong to this commit.

Reply:
mixed up patches. It was a PoC anyway